provisioning/miniprov/hare_mp/store.py

```diff
@@ -63,13 +66,30 @@ def get_cluster_id(self) -> str:
         cluster_id = self.get(f'server_node>{machine_id}>cluster_id')
         return cluster_id
 
-    def get_storage_set_id(self) -> int:
+    def get_storage_set_id(self) -> str:
         machine_id = self.get_machine_id()
         storage_set_id = self.get((f'server_node>{machine_id}>'
                                    f'storage_set_id'))
-        return int(storage_set_id)
+        return storage_set_id
 
     def get_hostname(self) -> str:
         machine_id = self.get_machine_id()
         hostname = self._raw_get(f'server_node>{machine_id}>hostname')
         return hostname
+
+    def get_no_of_nodes_in_ss(self) -> int:
+        cluster_id = self.get_cluster_id()
+        no_of_nodes_in_ss = 0
+        storage_set_id = self.get_storage_set_id()
+        i = 0
+
+        for storage_set in self.get(f'cluster>{cluster_id}>storage_set'):
```

Comment:
Can be refactored to a separate function like `get_storage_set_index_by_name(self, name: str) -> int`

Reply:
Done